language/extensions/move-table-extension/sources/Table.move

```diff
@@ -77,7 +77,7 @@ module extensions::table {
         val
     }
 
-    /// Returns true iff `table` contains an entry for `key`.
+    /// Returns true if `table` contains an entry for `key`.
```

Comment:
Actually `iff` is a shortcut for `if and only if`, which is commonly used in the formal verification world. But I'm OK with changing this if it is confusing ;-)

Reply:
Got it.